curriculum/challenges/english/25-front-end-development/quiz-recursion/66edd43cded6bff30944b676.md

```diff
@@ -124,86 +124,6 @@ A `Too much recursion` error.
 
 #### --text--
 
-Given the following recursion:
-
-```javascript
-function sum(n) {
-  if (n === 0) return 0;
-  return n + sum(n - 1);
-}
-sum(3);
-```
-
-Which of the following is the order of the function calls in the call stack?
-
-#### --distractors--
-
-```js
-sum(0);
-sum(1);
-sum(2);
-sum(3);
-```
-
----
-
-```js
-sum(1);
-sum(2);
-sum(3);
-```
-
----
-
-```js
-sum(3);
-sum(2);
-sum(1);
-```
-
-#### --answer--
-
-```js
-sum(3);
-sum(2);
-sum(1);
-sum(0);
-```
-
-### --question--
-
-#### --text--
-
-What will this code print?
-
-```javascript
-function sayHello(n) {
-  if (n <= 0) return [];
-  return [`Hello ${n}`, ...sayHello(n - 1)];
-}
-console.log(sayHello(3));
-```
-
-#### --distractors--
-
-"Hello 3"
-
----
-
-An error
-
----
-
-`["Hello 3", "...Hello"]`
-
-#### --answer--
-
-`["Hello 3", "Hello 2", "Hello 1"]`
-
-### --question--
-
-#### --text--
-
 How many times will the function call itself?
```

Comment:
```suggestion
How many times will the `mystery` function be called?
```
I agree with Tom. It should be clear that the initial call is included in the count.